tests/test_issues.py

```diff
@@ -76,7 +76,7 @@ def test_issue_8(self):
         yield from c.execute("drop table if exists test")
         yield from c.execute("""CREATE TABLE `test` (`station` int(10) NOT
             NULL DEFAULT '0', `dh`
-            datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+            datetime NOT NULL DEFAULT '1000-01-01 00:00:00',
```

Comment:
According to [MySQL docs](https://dev.mysql.com/doc/refman/5.7/en/datetime.html), supported range for dates is `1000-01-01` to `9999-12-31`.